pkg/workloads/console/controller.go

```diff
@@ -436,7 +495,7 @@ func (r *reconciler) updateStatus(job *batchv1.Job) error {
 	return nil
 }
 
-func calculateStatus(csl *workloadsv1alpha1.Console, job *batchv1.Job, pod *corev1.Pod) workloadsv1alpha1.ConsoleStatus {
+func calculateStatus(csl *workloadsv1alpha1.Console, authorised bool, job *batchv1.Job, pod *corev1.Pod) workloadsv1alpha1.ConsoleStatus {
```

Comment:
Discussed briefly and can possibly see it would be nice to have along with some other info like time of authorisation on the console status, but for now going to leave as is. We can always easily add this later if we feel the need.